core-tests/src/test/java/org/mule/runtime/core/DefaultMuleMessageSerializationTestCase.java

```diff
@@ -30,7 +32,8 @@
     @Test
     public void testSerializablePayload() throws Exception
     {
-        MuleMessage message = new DefaultMuleMessage(TEST_MESSAGE, muleContext);
+        // TODO MULE-9856 Replace with the builder
```

Comment:
Are there uses of this MutableMuleMessage that will not be replaced by the builder? If there aren't it would probably be best to avoid repeating the TODO and just have it in MutableMuleMessage since all it's uses are the ones that should be replaced.